content/participants/ttjc@rohankokane/index.md

```diff
@@ -0,0 +1,84 @@
+---
+title: rohankokane
+type: "participant"
+name: "Rohan Kokane"
+portfolio: "https://rohankokane.netlify.app/"
+github-link: "https://github.com/rohankokane"
+linkedin-link: "https://www.linkedin.com/in/rohankokane/"
+looking-for: "internship"
+description: "Intermediate in HTML CSS JS PHP and SQL | Learning advance JS  | Python | C | git"
+---
+
+Hi! I am Rohan Kokane, a tech enthusiast really loves building web projets where I can instantly see things building up.
+
+### introduction
+
+Now I am in Third Year of bachelors in Computer Engineering.
+
+## why are you taking this challenge?
+
+Actually I tried to register for Tanay Jobchallenge 2-3 months ago but couldn't register due to some technical difficulties and then I got busy into my college things and some projects.
+Today from Tanay's instagram I got to know its the last day to register and I thought lets give it another shot.
+
+So here I am, I want to get a real industrial intership where I can push myself everyday to build new real word projects and learn new tech.
+I have been working on college projects since few months but there are lot of restrictions to work on like using old stack and methods.
+
+## something interesting about you?
+
+I like to watch old classic movies. Sometimes like to write a litte bit and I have good communcation and leadership skills.
+
+### help I need?
+
+I need a mentor, someone who would just show me the right path and I am ready work hard and go up ladder of success.
+
+### how to contact me?
+- [Telegram](https://t.me/rohankokane)
+- [LinkedIn](https://www.linkedin.com/in/rohankokane/)
+- [Twitter](https://twitter.com/rohankokane19)
+
+
+### projects
+
+My projects: 
+
+### Covid-19 Awareness Portal
+
+_description_ In this global pandemic situation Let's be aware and not scared! A complete responsive web application where our college could conduct various activities for students in this quarantine. Now there is pictorial awareness quiz upon which can also earn certificates and it will be mailed by PHP mailer. A Wall of Unsung Heores, the people who are helping the needy ones through their small acts of kindness can share there stories and ideas by uploading on this wall. A small effort to spread positivity amid this lockdown.
+
+_stack_ Frontend with HTML, CSS and Javascript and PHP for backend with mySQL for Database.
+
+_hosted link_ https://www.rait.ac.in/covid19portal/
+
+_github link_ https://github.com/rohankokane/covid19portal
+
+### To do list
+
+_description_   ): A to-do application helps in organizing work better, where we can add list of tasks to do and upon completion can mark it as completed and also we can clear a particular task as well as the entire list. Built this one to learn Javascript 
+
+_stack_ HTML, CSS and Javascript
+
+_hosted link_ https://todolistbyrohan.netlify.app/
+
+_github link_ 
```

Comment:
provide link to the source code

Reply:
Hey!
I have provided the github link,
Kindly accept my request.